datalad_gooey/app.py

```diff
@@ -295,6 +302,39 @@ def _get_info(self):
               'datalad.org</a>.'
         mbox(self.main_window, title, msg)
 
+    def _get_diagnostic_info(self):
+        self.execute_dataladcmd.emit(
+            'wtf',
+            MappingProxyType(dict(
+                result_renderer='disabled',
+                on_failure='ignore',
+                return_type='generator',
+            )),
+            MappingProxyType(dict())
+        )
+        self._cmdexec.results_received.connect(
+            self._app_cmdexec_results_handler)
+
+    @Slot(Interface, list)
+    def _app_cmdexec_results_handler(self, cls, res):
+        for r in res:
```

Comment:
Make this conditional on `cls == Wtf`, instead of rejecting each and every result. This is faster. Keep in mind that any command execution result will come in here. Not just `wtf`.

Reply:
thx!